molecule/default/tasks/patched.yml

```
@@ -0,0 +1,107 @@
+---
+- block:
+    - set_fact:
+        patch_only_namespace:
+          first: patched-namespace-1
+          second: patched-namespace-2
+
+    - name: Ensure namespace {{ patch_only_namespace.first }} exist
+      kubernetes.core.k8s:
+        definition:
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+            name: "{{ patch_only_namespace.first }}"
```

Could I ask that this test check to ensure that other parts of the resource are not touched (i.e. left as-is). For example, adding a label and annotation and then later testing that that label and annotation still exist after the patch:

```suggestion
            name: "{{ patch_only_namespace.first }}"
            labels:
              existingLabel: "labelValue"
            annotations:
              existingAnnotation: "annotationValue"
```